langs/julia.yaml

```diff
@@ -4,12 +4,12 @@ aliases:
 name: "Julia"
 
 install:
-  prepare:
-    apt:
-      - julia
-  apt:
-    - julia
   manual: |
+    wget https://julialang-s3.julialang.org/bin/linux/x64/1.7/julia-1.7.0-linux-x86_64.tar.gz
```

Comment:
Can we update this to automatically detect the latest version, as suggested in https://github.com/raxod502/riju/blob/master/doc/tutorial/install.md?